index.html

```diff
@@ -326,26 +331,35 @@ <h2><dfn>observe()</dfn> method</h2>
         <p>The <a>observe()</a> method instructs the user agent to
         <dfn>register the observer</dfn> and must run these steps:</p>
         <ol data-link-for="PerformanceObserverInit">
-          <li>Let <var>entry types</var> be <var>options</var> <a>entryTypes</a>
-            sequence.</li>
-          <li>Remove all unsupported types from <var>entry types</var>. The user
-            agent SHOULD notify developers if <var>entry types</var> is modified.
-            For example, a console warning listing removed types might be appropriate.</li>
-          <li>If the resulting <var>entry types</var> sequence is an empty sequence,
+          <li>Set <a>observed entry types</a> to a copy of <var>options</var>'
+            <a>entryTypes</a> sequence.
+          </li>
+          <li>For each key <var>entryType</var> in the <var>options</var> dictonary that
+            is not <a>entryTypes</a>, add the string <var>entryType</var> to
+            <a>observed entry types</a>.
+          </li>
+          <li>Remove all unsupported types from <a>observed entry types</a>. The user
+            agent SHOULD notify developers if <a>observed entry types</a> is modified.
+            For example, a console warning listing removed types might be appropriate.
+          </li>
+          <li>If <a>observed entry types</a> is an empty sequence,
             abort these steps. The user agent SHOULD notify developers when the
             steps are aborted to notify that registration has been aborted. For
-            example, a console warning might be appropriate.</li>
+            example, a console warning might be appropriate.
+          </li>
           <li>If the <a>list of registered performance observer objects</a> of
             <a>relevant global object</a> contains a <a>registered performance
-            observer</a> whose `observer` is the <a>context object</a>, replace its `options`, with <var>options</var>.
+            observer</a> whose `observer` is the <a>context object</a>, replace its
+            `options`, with <var>options</var>.
           </li>
          <li>Otherwise, append a new <a>registered performance observer</a>
           object to the <a>list of registered performance observer objects</a> of
           <a>relevant global object</a>, with the <a>context object</a> as
           `observer` and <var>options</var> as the `options`.
           </li>
-          <li>If <var>options</var>' <a>buffered</a> flag is set, for each
-            <var>entry type</var> in <var>entry types</var> sequence:
+          <li>For each <var>entry type</var> in <a>observed entry types</a>, if the
+            <a data-link-for="PerformanceEntryObserverOptions">buffered</a> flag is set
+            in <var>options</var>' dictionary named <var>entry type</var>:
```

Comment:
`options` is a `PerformanceObserverInit`, right? Or is it `PerformanceEntryObserverOptions`?

Reply:
Yes, but `buffered` would be inside the `PerformanceEntryObserverOptions` corresponding to that entry type (if it exists), which is itself inside the `PerformanceObserverInit` `options`. I moved this flag so that it can be an entryType-specific parameter.